examples/react-demo/src/App.tsx

```diff
@@ -51,11 +51,11 @@ function App({
             : undefined
         }
       />
-      <h5 className="font-semibold mt-8">
-        2. Click Connect Wallet button (or disconnect and reconnect)
+      <h5 className="text-xl mt-[30px] mb-[25px]">
+        Step 2. Click Connect Wallet button (or disconnect and reconnect)
       </h5>
-      <h5 className="font-semibold mb-8">
-        3. Interact with other smart contract canisters on behalf of your connected user
```

Comment:
Step 3. Interact with other smart contract canisters on behalf of your connected user